resources/assets/js/features/public-profiles/components/DriverRating.js

```diff
@@ -20,69 +20,64 @@ class DriverRating extends React.Component {
                     <div className="col-sm-6">
                         <h5>{translate('driver_public_profile.driver_rating_breakdown')}</h5>
                         <div className="pull-left">
-                            <div className="pull-left" style={{width: '35px', lineHeight: 1}}>
-                                <div style={{height:'9px', margin: '5px 0'}}>5</div>
+                            <div className="pull-left rating-marker-outer">
```

Comment:
I already described class naming concept that will help avoid any problems in future
rating__marker rating-marker can be used here instead of outer/inner